examples/1.1-check-solarium-and-ping.php

```diff
@@ -1,10 +1,11 @@
 <?php
 
-require(__DIR__.'/init.php');
-htmlHeader();
+require(__DIR__ . '/vendor/autoload.php');
```

Comment:
As this is an example maybe better not uss `__DIR__` but

```php
require('./vendor/autoload.php');
```
then people can copy/paste and maybe run immediately.